components/datatypes/agentic/conversation.schema.json

```diff
@@ -0,0 +1,188 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/agentic/conversation",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Conversation",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Construct to represent agents, agent skills, etc.",
```

Comment:
@rbuck23 it would be good to add which fields are required, the examples help, but it is not clear which fields are mandatory.